Okane.Application/Expenses/Response.cs

```diff
@@ -1,3 +1,3 @@
 namespace Okane.Application.Expenses;
 
-public record Response(int Id, int Amount, string Category);
\ No newline at end of file
+public record Response(int Id, int Amount, string Category, string Description);
```

Comment:
Como description is opcional, nos conviene que se nullable.

`string? Description`